app/views/projects/index.html.erb

```diff
@@ -100,6 +101,12 @@ See docs/COPYRIGHT.rdoc for more details.
                 <% end %>
                 <%= link_to_project(project, {}, {}, false) %>
               </td>
+              <td>
+                <% if project.status.try(:code) %>
+                  <span class="project-status--bulb -inline <%= project_status_css_class(project.status) %>"></span>
+                  <span class="project-status--name <%= project_status_css_class(project.status) %>"><%= project_status_name(project.status) %></span>
+                <% end %>
+              </td>
```

Comment:
You need to increase the colspan of the description row [here](https://github.com/opf/openproject/blob/dev/app/views/projects/index.html.erb#L147). Otherwise the border is too short.
<img width="370" alt="Bildschirmfoto 2019-10-17 um 09 15 01" src="https://user-images.githubusercontent.com/7457313/66986538-6676eb00-f0bf-11e9-893c-de7c17cbc960.png">